bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/DefaultEntryLogger.java

```diff
@@ -1086,7 +1087,9 @@ EntryLogMetadata extractEntryLogMetadataFromIndex(long entryLogId) throws IOExce
                 bc.read(sizeBuffer.get(), offset);
 
                 int ledgersMapSize = sizeBuffer.get().readInt();
-
+                if (ledgersMapSize <= 0) {
```

Comment:
why are we adding this check ?

Reply:
If there are not ledgers in ledgerMap, we just skip the following reading.